Framework/CMakeLists.txt

```diff
@@ -144,7 +144,7 @@ add_subdirectory (ScriptRepository)
 ###########################################################################
 
 set ( FRAMEWORK_LIBS Kernel HistogramData Indexing Beamline Geometry API DataObjects
-                     PythonKernelModule PythonGeometryModule PythonAPIModule
+                     PythonInterface PythonKernelModule PythonGeometryModule PythonAPIModule
```

Comment:
Can we remove `PythonKernelModule PythonGeometryModule PythonAPIModule PythonDataObjectsModule PythonCurveFittingModule`? In theory `PythonInterface` is supposed to depend on all of those.